www/class/centreonSession.class.php

```diff
@@ -68,6 +68,13 @@ public static function restart()
         session_regenerate_id(true);
     }
 
+    public function writeSessionClose($key, $value)
```

Comment:
No need to create a specific method into the centreonSession.class.php
You can move it directly into the centreon_topcounter.class.php

Reply:
it won't be clean i think